src/neuroglancer/ui/layer_side_panel.css

```diff
@@ -15,7 +15,7 @@
  */
 
 .neuroglancer-layer-side-panel {
-  width: 290px;
+  width: 304px;
```

Comment:
Still a bit too short to hold a maximum sized `Uint64` (20 characters) plus its hide/copy buttons. Favorable solution however would be the use of pictographs as discussed.